sdcflows/interfaces/epi.py

```diff
@@ -37,6 +37,8 @@
 class _GetReadoutTimeInputSpec(BaseInterfaceInputSpec):
     in_file = File(exists=True, desc="EPI image corresponding to the metadata")
     metadata = traits.Dict(mandatory=True, desc="metadata corresponding to the inputs")
+    use_estimate = traits.Bool(False, desc='Use "Estimated*" fields to calculate TotalReadoutTime')
```

Comment:
If I understand Chris Rorden's thinking, this perhaps is best if enabled?

```suggestion
    use_estimate = traits.Bool(True, desc='Use "Estimated*" fields to calculate TotalReadoutTime')
```

Reply:
My thought was that since this isn't BIDS, using these fields should be opt-in instead of default.

Reply:
```suggestion
    use_estimate = traits.Bool(False, usedefault=True, desc='Use "Estimated*" fields to calculate TotalReadoutTime')
```